diff --git a/bigtop-deploy/vm/vagrant-puppet-openstack/para-provision.sh b/bigtop-deploy/vm/vagrant-puppet-openstack/para-provision.sh
index 96f4b1a..7b191e8 100755
--- a/bigtop-deploy/vm/vagrant-puppet-openstack/para-provision.sh
+++ b/bigtop-deploy/vm/vagrant-puppet-openstack/para-provision.sh
@@ -85,7 +85,7 @@ done | parallel_provision
 echo "preparing for smoke tests..."
 if [ "$RUN_SMOKE_TESTS" = "true" ]; then
     echo "running smoke tests..."
-    vagrant ssh hadoop-bigtop-para$NUM_INSTANCE -c "sudo su <<HERE
+    vagrant ssh hadoop-bigtop$NUM_INSTANCE -c "sudo su <<HERE
     cd /bigtop-home/bigtop-tests/smoke-tests
     export HADOOP_CONF_DIR=/etc/hadoop/conf/
     export BIGTOP_HOME=/bigtop-home/
